marimo/_messaging/ops.py

```diff
@@ -321,6 +324,19 @@ def broadcast_stale(
     ) -> None:
         CellOp(cell_id=cell_id, stale_inputs=stale).broadcast(stream)
 
+    @staticmethod
+    def broadcast_serialization(
+        cell_id: CellId_t,
+        serialization: TopLevelStatus,
+        stream: Stream | None = None,
+    ) -> None:
+        status = (
```

Comment:
could we make this hint typesafe (e.g `Literal[]`)?

Reply:
It's a dynamic value since the hint can be dependent on the refs etc... But I put something in to indicate it's from a static set of values, and mypy didn't seem to care.